src/Nethermind/Nethermind.Core/ClientVersion.cs

```diff
@@ -33,8 +33,12 @@ static ClientVersion()
             }
             string date = DateTime.Now.ToString("yyyyMMdd");
             string gitTag = File.Exists(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "git-hash")) ? File.ReadAllText(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "git-hash")).Trim().Replace("g", "") : string.Empty;
-
-            Description = $"Nethermind/v{gitTag}-{date}/{RuntimeInformation.OSArchitecture}-{osDescription}/{RuntimeInformation.FrameworkDescription.Trim().Replace(".NET ", "").Replace(" ", "")}";
+            string gitBranch = File.Exists(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "git-branch")) ? File.ReadAllText(Path.Combine(AppDomain.CurrentDomain.BaseDirectory, "git-branch")).Trim().Replace("/", "-") : string.Empty;
+            if (gitBranch != "master")
+            {
+                gitTag = gitTag.Insert(5, "b");
```

Comment:
b did not mean branch - it meant the next version - so probably better to be able to release from a tag on the branch
so we can create a hotfix branch of the 1.4.1 tag and tag it 1.4.1b and then version is picked as 1.4.1b